testnet_nodes.yml

```
@@ -0,0 +1,83 @@
+      private_key: cP5Qb4riwGxj7qGgXKKWffrntJwDzkn49iQv569H25CcRqQqSpEh
+    operator:
+      public_key: >-
+        b44cee83a79fa151527e527f3f4f5ba022e73ae8b0d913c4185a45c2a129aef935a585a7a725edcb36ece72a95758688
+      private_key: 05a2cf8c9e2771ab0f4189bc7bf8eaf56fc3b1deb376825848d6e3e847b5b3e4
+    voter:
+      address: yhooszW6XxCpVeZSyik75LMHdsahRJJwos
+      private_key: cTQFdQVdLM41opTs4D2XWPT4vz2yTpF7NUA43XUrFAgftYZivaTY
+    node_key:
+      id: f460a07a2b0dc932e1d8ff2dee2b984ac415dceb
+      private_key: >-
+        5KGIQTFmtygGAzRMc/YHi50YJ95E+3Ro+8Xm6F1+ZMZN9nedIOxWx0B9t2cMl+a5lQ5fO4smxY7N/2Qno0WOrg==
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)